CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 2.0.0
+
+Breaking changes
+
+- ALL: Minimum Node.js version is now 10.13.
```

Comment:
Can we add a note on browser support?

Reply:
Hmm, no `BigInt` support in Edge, IE or Safari right now according to MDN. Is it definitely the right time for this?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt#Browser_compatibility

Reply:
The only browser environments we that are important for us right now are the ones that suport chrome-like browser extensions. So I think modern Chromium-based browsers and Firefox is sufficient.

As far as I know, Microsoft is already planning to build a browser based on Chromium. The only use case for Safari I see is the token finder and a potential block explorer, which will not be out before summer.

So I think it is a far trade-off to require BigInt support.

Reply:
Thanks for checking this by the way!